packages/multi-trait-rubric/configure/src/index.js

```diff
@@ -40,13 +46,15 @@ export default class MultiTraitRubricElement extends HTMLElement {
       }
 
       (scale.traits || []).forEach(trait => {
-        trait.scorePointsDescriptors = [ ...trait.scorePointsDescriptors ];
+        if (!trait) return;
```

Comment:
Let's make sure we actually update the model and make it valid, Returning undefined here will break the code.